redfish-core/lib/pcie.hpp

```diff
@@ -167,7 +167,11 @@ inline void requestRoutesSystemPCIeDevice(App& app)
                             &pcieDevProperties["DeviceType"]);
                         property)
                     {
-                        asyncResp->res.jsonValue["DeviceType"] = *property;
+                        std::string deviceType = analysisGeneration(*property);
```

Comment:
I think you need a new analysis function here?  analysisDeviceType() maybe?

Reply:
It appears to be a mistake. We do not need any analysis for "DeviceType".
Thanks for pointing this out. I will fix it.